src/router/loaders.tsx

```diff
@@ -419,24 +421,17 @@ export const stepUfoeretrygdAFPAccessGuard =
     const state = store.getState()
     const afp = selectAfp(state)
     const foedselsdato = selectFoedselsdato(state)
-    const getLoependeVedtakResponse =
-      apiSlice.endpoints.getLoependeVedtak.select(undefined)(state)
+    if (!foedselsdato) throw new Error('Missing foedselsdato')
+    const loependeVedtak =
+      apiSlice.endpoints.getLoependeVedtak.select()(state).data
+    if (!loependeVedtak) throw new Error('Missing loependeVedtak')
```

Comment:
I stedet for å kaste feil hvis denne ikke finnes i state allerede så kan man brukes rtk-query for å prøve å hente den på nytt (gitt den ikke er hentet) og la det promiset blir rejected

```javascript
const loependeVedtak = await store
    .dispatch(apiSlice.endpoints.getLopenendevedtak.initiate())
    .unwrap()
```